src/main/java/com/xpanxion/java/springboot/da1/demo/model/student6/Gym.java

```diff
@@ -0,0 +1,90 @@
+package com.xpanxion.java.springboot.da1.demo.model.student6;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="GYM")
+public class Gym {
```

Comment:
Can you please change this from `Gym` to `Gym6`?

Reply:
sure thing should i do the same with service, repo etc?

Reply:
Yes, please add your student id to anything that has a SpringBoot annotation.